src/main/java/dk/kb/present/PresentFacade.java

```diff
@@ -181,7 +206,8 @@ private static StreamingOutput getRecordsFull(
 
     // Direct ds-storage record JSON
     private static StreamingOutput getRecordsSolr(DSCollection collection, Long mTime, Long maxRecords,
-                                                  HttpServletResponse httpServletResponse) {
+                                                  HttpServletResponse httpServletResponse,
+                                                  Function<List<DsRecordDto>, Stream<DsRecordDto>> accessPredicate) {
```

Comment:
All other methods calls the function accessFilter. Maybe choose one or the other?

Reply:
Leftover from previous code. Nice catch.